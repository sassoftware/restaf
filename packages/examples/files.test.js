let testFunctions = require("./testFunctions");
let testInfo;
beforeAll(async () => {
	testInfo = await require('./lib/setupAll')();
});
test("files: create and retrieve content", async () => {
	expect.assertions();
    let l = await testFunctions.filesCreate(testInfo);
    expect(l).toMatchSnapshot();
});


test('files - paginate thru files', async () => {
	expect.assertions();
	let l = await testFunctions.filesPaginate(testInfo);
	expect(l).toMatchSnapshot();
});