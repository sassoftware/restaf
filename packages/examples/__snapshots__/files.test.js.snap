// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`files - paginate thru files 1`] = `"done"`;

exports[`files: create and retrieve content 1`] = `
Object {
  "me": "hi there",
}
`;
